src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -227,7 +227,11 @@ bool JwtAuthenticator::OkToBypass() {
 void JwtAuthenticator::DoneWithStatus(const Status& status) {
   ENVOY_LOG(debug, "Jwt authentication completed with: {}",
             JwtAuth::StatusToString(status));
```

Comment:
Could you please modify the ENVOY_LOG here to add allow_missing_or_failed information, so that it would be easier for debugging when allow_missing_or_failed is true and status is not Status::OK?

Reply:
Done.